### DISCUSSION TITLE
Comparing Multiple Species in Metaboverse

### DISCUSSION THREAD

**Discussion by BrooksLeitner** [Metaboverse/Metaboverse#103] [Q&A]
Is a possibility to compare similarities between multiple species in metaboverse? I have 5 datasets of different species and am curious which metabolic pathways are best conserved between species and in different tissues, and wondered if metaboverse would be a good way to approach this question. 

Thanks!

**Comment by j-berg** [comment]
Hi @bleitner-exercise, sorry for the delayed response. 

Currently, Metaboverse is not designed to be able to handle an analysis like you propose. Depending on your input datasets, however, you could input your different species' datasets as a multi-condition dataset and choose one organism to act as the top-level network to map your data across species to. This would work best if you are just dealing with metabolomics data. If you want to consider transcriptomics or proteomics datasets, this will be more complicated as you would need to be able to map gene/protein IDs between species and the top-level species you use for modeling. Feel free to email your data to me at jordan.berg <at> biochem.utah.edu if you would like me to play around and help with formatting if you think this type of analysis would be helpful.

**Reply by BrooksLeitner** [reply]
Thanks! I'm working on another project in the mean time but when I circle back I'll definitely get in touch. I am working with transcriptomic data and it certainly has been pretty annoying to match everything up.
